integration_tests/support/delay.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-restricted-syntax */
+
+// Set COMMAND_DELAY above zero for demoing to stakeholders,
+const COMMAND_DELAY = Cypress.env('COMMAND_DELAY') || 0
```

Comment:
Genuine question, not a veiled suggestion: should this be namespaced to `CYPRESS_COMMAND_DELAY`?

Reply:
Good point, I'll update this